app/design/adminhtml/default/default/template/ebizmarts/mailchimp/system/config/form/field/array_dropdown.phtml

```diff
@@ -14,28 +14,28 @@ if (!$this->_addAfter) {
 $colspan = $colspan > 1 ? 'colspan="' . $colspan . '"' : '';
 ?>
 
-<div class="grid" id="grid<?php echo $this->quoteEscape($htmlId) ?>">
+<div class="grid" id="grid<?php echo $this->gethelper()->mcEscapeQuote($htmlId) ?>">
     <table cellpadding="0" cellspacing="0" class="border">
         <tbody>
 
-        <tr class="headings" id="headings<?php echo $this->quoteEscape($htmlId) ?>">
+        <tr class="headings" id="headings<?php echo $this->gethelper()->mcEscapeQuote($htmlId) ?>">
             <?php foreach ($this->_columns as $columnName => $column): ?>
-                <th><?php echo $this->quoteEscape($column['label']) ?></th>
+                <th><?php echo $this->gethelper()->mcEscapeQuote($column['label']) ?></th>
             <?php endforeach; ?>
-            <th <?php echo $this->quoteEscape($colspan) ?>></th>
+            <th <?php echo $this->gethelper()->mcEscapeQuote($colspan) ?>></th>
         </tr>
 
-        <tr id="addRow<?php echo $this->quoteEscape($htmlId) ?>">
-            <td colspan="<?php echo $this->quoteEscape(count($this->_columns)) ?>">
+        <tr id="addRow<?php echo $this->gethelper()->mcEscapeQuote($htmlId) ?>">
+            <td colspan="<?php echo $this->gethelper()->mcEscapeQuote(count($this->_columns)) ?>">
                 <button style=""
                         onclick=""
                         class="scalable add"
                         type="button"
-                        id="addNewType<?php echo $this->quoteEscape($htmlId) ?>">
+                        id="addNewType<?php echo $this->gethelper()->mcEscapeQuote($htmlId) ?>">
                     <span>
                         <span>
                             <span>
-                                <?php echo $this->quoteEscape(
+                                <?php echo $this->gethelper()->mcEscapeQuote(
```

Comment:
This can be more straight forward by calling only one function that calls the helper internally.